assets/javascripts/discourse/components/discourse-post-event/invitee.gjs

```diff
@@ -0,0 +1,65 @@
+import Component from "@glimmer/component";
+import { concat } from "@ember/helper";
+import { service } from "@ember/service";
+import { htmlSafe } from "@ember/template";
+import { eq } from "truth-helpers";
+import AvatarFlair from "discourse/components/avatar-flair";
+import concatClass from "discourse/helpers/concat-class";
+import { formatUsername } from "discourse/lib/utilities";
+import { avatarImg } from "discourse-common/lib/avatar-utils";
+
+export default class DiscoursePostEventInvitee extends Component {
+  @service site;
+  @service currentUser;
+
+  get statusIcon() {
+    let statusIcon;
+    switch (this.args.invitee.status) {
+      case "going":
+        statusIcon = "fa-check";
+        break;
+      case "interested":
+        statusIcon = "fa-star";
+        break;
+      case "not_going":
+        statusIcon = "fa-times";
+        break;
+    }
+    return statusIcon;
+  }
+
+  get avatarImage() {
+    return htmlSafe(
+      avatarImg({
+        avatarTemplate: this.args.invitee.user.avatar_template,
+        size: this.site.mobileView ? "tiny" : "large",
+        title: this.args.invitee.user.name
+          ? formatUsername(this.args.invitee.user.name)
+          : this.args.invitee.user.username,
+      })
+    );
+  }
+
+  <template>
+    <li
+      class={{concatClass
+        "event-invitee"
+        (if @invitee.status (concat "status-" @invitee.status))
+        (if (eq this.currentUser.id @invitee.user.id) "is-current-user")
+      }}
+    >
+      <a class="topic-invitee-avatar" data-user-card={{@invitee.user.username}}>
+        {{this.avatarImage}}
```

Comment:
we could also use here the [avatar helper](https://github.com/discourse/discourse/blob/21f23cc0328640fd87e30fb3a4b4d62960a8aa43/app/assets/javascripts/discourse/app/helpers/bound-avatar-template.js) cvx suggested